examples/counter/crate/src/lib.rs

```diff
@@ -28,59 +28,53 @@ impl Render for Counter {
     {
         let val = bumpalo::format!(in bump, "{}", self.val);
 
-        let increment = Node::element(
-            bump,
-            "button",
-            [Attribute {
-                name: "data-action",
-                value: "increment",
-            }],
-            [Node::text("+")],
-        );
-
-        let decrement = Node::element(
-            bump,
-            "button",
-            [Attribute {
-                name: "data-action",
-                value: "decrement",
-            }],
-            [Node::text("-")],
-        );
-
         Node::element(
             bump,
             "div",
             [],
-            [increment, Node::text(val.into_bump_str()), decrement],
+            [],
+            [
+                Node::element(
+                    bump,
+                    "button",
+                    [on(bump, "click", |root, vdom, _event| {
```

Comment:
Would it be possible to do something like `bump.on("click", ...)`? Maybe via something like a newtype wrapper or the like?

Reply:
I was thinking of adding an extension trait, and also thinking of adding `bump.div(...)` as shorthand for `Node::element(bump, "div", ...)`. Was wondering if that was a little too magicky or not though. Also, just generally not too focused on ergo yet, and hoping I can mostly kick that down the road to something like the `typed-html` crate.